cl/search/factories.py

```diff
@@ -33,7 +34,6 @@ class CourtFactory(DjangoModelFactory):
     class Meta:
         model = Court
 
-    id = Faker("random_id")
```

Comment:
After all I said about removing IDs, this one *might* need to stay. I'm not sure. Courts are the one exception to this rule because they don't use sequential numbers. Could you check if `make_dev_data` works with this line removed? I think it's going to fail because it relies on the ID getting created — but I'm just guessing.

Reply:
interestingly, I ran make dev data and returned 

`botocore.exceptions.NoCredentialsError: Unable to locate credentials`

Reply:
I think ive resolved this but I look forward your commentary